lib/connectors/base_client.go

```diff
@@ -42,6 +44,16 @@ func (c *baseClient) SetDebug() {
 	c.restClient.SetDebug(true)
 }
 
+func (c *baseClient) SetClientCertificates(certFile string, keyFile string) {
+	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
+
+	if err != nil {
+		log.Fatalf("ERROR client certificate: %s", err)
```

Comment:
lib should avoid logging or stopping the program.
It would be better to return the error, catch it in the cli part and then log it.